### ISSUE TITLE
`persistTab()` does not work with `spatie-laravel-translatable-plugin`

### ISSUE THREAD

**Issue by AlexisSerneels** [laravel-filament/filament#10278]
### Package

filament/filament

### Package Version

v3.1.17

### Laravel Version

v10.35

### Livewire Version

v3.3.0

### PHP Version

PHP 8.2.12

### Problem description

This bug reports relates to : https://github.com/filamentphp/filament/pull/10062#issuecomment-1849065816

Bug in video:

https://github.com/filamentphp/filament/assets/287688/a1b374cc-25c9-4fb2-9963-1c1cfcaa0cd1

Using `->persistTab()` when used in combination with `filamentphp/spatie-laravel-translatable-plugin` breaks the form after a page refresh.

Here is the reported JS error:

![CleanShot 2023-12-10 at 20 54 06@2x](https://github.com/filamentphp/filament/assets/287688/8e74696f-df61-4305-8927-3717d695f371)

It it because the value of the current tab is lost after a language switch and set to ``undefined`` in the LocalStorage.

![CleanShot 2023-12-10 at 20 55 03@2x](https://github.com/filamentphp/filament/assets/287688/8eda34aa-fafd-4888-9043-bf90560204aa)

After investigations, it seems that the value of `getTabs()` from the file `tabs.blade.php` returns an empty array (Line 50) when the language is switched. 

I guess the hook ```Livewire.hook``` introduced in https://github.com/filamentphp/filament/pull/10062/commits/1cbaca307089fc9bbfc95f11beaaff541260a244 is not performing well with language switch.

### Expected behavior

It's expected that the current tab value is persisted in the localStorage, not `undefined` and that the form is shown.

### Steps to reproduce

1. Pull the issue repository.
2. Migrate
3. Seed
4. Visit the ProductResource in Filament
5. Select a tab
6. Change locale
7. Refresh the page

### Reproduction repository

https://github.com/AlexisSerneels/filament-bug

### Relevant log output

```shell
Uncaught SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
    storageGet http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:5691
    persist http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:5657
    initialize http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:908
    recurse http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:892
    recurse http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:887
    initInterceptors2 http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:900
    <anonymous> http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:3239
    flushHandlers http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:1135
    stopDeferring http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:1140
    deferHandlingDirectives http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:1143
    initTree http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:654
    start http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:609
    start http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:608
    start2 http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:8283
    <anonymous> http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:9401
    EventListener.handleEvent* http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:9400
    <anonymous> http://127.0.0.1:8000/livewire/livewire.js?id=8a199ab2:9404
```

**Comment by danharrin** [comment]
I think this was probably fixed in v3.1.19.

**Comment by danharrin** [comment]
Let me know if not and I will reopen
